app/scripts/proactive/templates/file-browser-template.html

```diff
@@ -47,27 +50,36 @@ <h5>
                <table style="width: 100%; table-layout:fixed;">
                    <thead>
                         <th>Name</th>
+                        <th width="80em">Rights</th>
                         <th width="80em">Size</th>
                         <th width="120em">Type</th>
                         <th width="160em">Modified</th>
                    </thead>
                    <tbody id="files-tbody">
-                    <% for (var index in directories) { %>
+                    <% for (var index in directories) {
+                        if (directories[index].name.indexOf('.') != 0 || showHidden) {
+                    %>
                     <tr class="file-browser-dir-tr">
                         <td class="file-browser-dir" value="<%- currentPath %><%- directories[index].name %>/"> <i class="far fa-folder"> </i> <%- directories[index].name %> </td>
+                        <td> <%- directories[index].rights %> </td>
                         <td> </td>
                         <td> Folder </td>
                         <td> <%- directories[index].modified %> </td>
                     </tr>
-                    <% } %>
-                    <% for (var index in files) { %>
+                    <%   }
+                       } %>
+                    <% for (var index in files) {
```

Comment:
I don't think you need to close then open the `%>` and `<%` tokens here.

Reply:
yes, right